chemberta/utils/raw_text_dataset.py

```diff
@@ -108,16 +107,64 @@ def __getitem__(self, i):
         feature_dict = self.dataset[i]
         example = self.preprocess(feature_dict)
         return example
-    
-    
+
+
+class RegressionTextDataset(Dataset):
+    def __init__(self, tokenizer, file_path: str, block_size: int):
+        print("init dataset")
+        self.tokenizer = tokenizer
+        self.file_path = file_path
+        self.block_size = block_size
+
+        data_files = get_data_files(file_path)
+        self.dataset = load_dataset("text", data_files=data_files)["train"]
+        example_line = self.dataset[0]
```

Comment:
this is fine but we probably want to add a quick note that structure is inferred from the first line